src/pages/scripts/tutorials/ids-object-model/index.md

```diff
@@ -0,0 +1,40 @@
+# UXP Scripts for InDesign Server
+Since InDesign v18.0, InDesign Server supports UXP Scripts as well.
+
+
+For running UXP Scripts in InDesign Server, you need both - InDesign and InDesign Server. It will be easier if you start by writing a script for InDesign and then make changes to the object model based on InDesign Server. Or else,  to see the result of your script in InDesign Server, you would have to save the document and open it using InDesign, or export the document and view the exported file in another program (like Acrobat ® or Adobe Reader®). If you use InDesign to develop your InDesign Server scripts, you can reduce your development time dramatically.
+
+<!-- TODO add installation steps to Introduction tab
```

Comment:
@anoopvalomkot  Pending item

Reply:
Please create a Jira ticket.

Reply:
@padmkris123 The documentation for InDesign Server is at https://helpx.adobe.com/in/indesign/using/indesign-server.html

We could just link to same. The distribution channel and installation of InDesign Server is a bit different, we can't cover it under UXP.